app/main.py

```diff
@@ -1,6 +1,35 @@
+import os
+from datetime import datetime as dt
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+BASE_URL = "https://api.weatherapi.com/v1/current.json?"
+CITY = "Paris"
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    api_key = os.getenv("API_KEY")
+    if not api_key:
+        raise ValueError("No API key found in environment variables")
+
+    url = BASE_URL + f"key={api_key}&q={CITY}"
```

Comment:
Code Style Issue: You are using string concatenation to create your URL. It would be better to use the `requests` library's parameters feature to pass in your parameters. This would make your code cleaner and easier to read.